Multitouch Support/Native/VoodooI2CNativeEngine.cpp

```diff
@@ -67,16 +67,15 @@ MultitouchReturn VoodooI2CNativeEngine::handleInterruptReport(VoodooI2CMultitouc
 
         // Force Touch emulation
         // The button state is saved in the first transducer
-        if (((VoodooI2CDigitiserTransducer*) event.transducers->getObject(0))->physical_button.value() && isForceClickEnabled()) {
+        if (firstTransducer->physical_button.value() && isForceClickEnabled()) {
```

Comment:
Should force touch emulation happen if just the secondary button is down?

Reply:
Force touch emulation shouldn't be used on touchpads with buttons. It's for touchpads with clickable surface.